### ISSUE TITLE
Bottom Navigation

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/cfgov-refresh#69]
Short description explaining the high-level reason for the new issue.

## Current behavior
![IMG_5ADC65FE0FB5-1](https://user-images.githubusercontent.com/22622384/85417608-d2a6ca00-b53d-11ea-83ad-f44a71cd8b54.jpeg)

## Expected behavior
----^ without grey line

## Steps to replicate behavior

1. Navigate to a new page
2. Start scrolling. Grey line appears at the bottom of the page around to the bottom navigation. It disappears few seconds after the scrolling stops.

**Comment by MissyM2** [comment]
@shubhi-raft can you give me a little more detail?  Did you test this on your computer or phone?  Also, did it happen consistently?  I am trying to replicate this but have not had any luck.

I am tagging the rest of the team.  Can you please test this and try to get it to replicate.  If so, send me the details:  what screens did it appear on?  Did it appear on multiple screens? Were you testing on phone/computer?  Please test on both.  This is a high priority. @RedNumberIX @LesliePerucci @hstout7369

**Comment by shubhi-raft** [comment]
test on a iphone. works fine on computer. @MissyM2 and yes, happens consistently.

**Comment by shubhi-raft** [comment]
i am not seeing this anymore. we can close this for now and check again once everything is merged.
